phpunit.xml.dist

```diff
@@ -30,6 +30,7 @@ It's auto-generated by sonata-project/dev-kit package.
 
     <php>
         <ini name="precision" value="8" />
+        <ini name="memory_limit" value="256M" />
```

Comment:
```
<!--
DO NOT EDIT THIS FILE!
It's auto-generated by sonata-project/dev-kit package.
-->
```

Reply:
Yes sorry, i will revert. I work on Symfony 4 retro compatibility and i will re-send PR.

Reply:
No problem ;)